dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -174,28 +247,44 @@ export const Columns: React.FC<{
 			)}
 
 			<li role="none">
-				<ThemeProvider theme={{ ...buttonThemeBrand }}>
-					<div css={searchBar}>
-						<LinkButton
-							href="https://www.google.co.uk/advanced_search?q=site:www.theguardian.com"
-							tabIndex={-1}
-							className="selectableMenuItem"
-							priority="secondary"
-							icon={
-								<SvgMagnifyingGlass
-									isAnnouncedByScreenReader={true}
-									size="medium"
-								/>
-							}
-							aria-label="Search with google"
-							data-link-name="nav2 : search : submit"
-							type="submit"
-						>
-							Search
-						</LinkButton>
-					</div>
-				</ThemeProvider>
+				<form css={searchBar} action="https://www.google.co.uk/search">
+					<TextInput
+						hideLabel={true}
+						label="Search input"
+						cssOverrides={searchInput}
+						name="q"
```

Comment:
Does this name represent the `q` query param sent to google in which case it _needs_ to be 'q'?

If so, can we add a comment explaining it just in case anyone lands here in the future thinking it needs renaming (which was my first instinct!)

If not, can we use a longer, more descriptive name?